deploy/Deploy.sol

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+pragma solidity ^0.8.13;
+
+import { ALMProxy }          from "src/ALMProxy.sol";
+import { ForeignController } from "src/ForeignController.sol";
+import { MainnetController } from "src/MainnetController.sol";
+import { RateLimits }        from "src/RateLimits.sol";
+
+library ALMProxyDeploy {
+
+    function deploy(address admin) external returns (address almProxy) {
+        almProxy = address(new ALMProxy(admin));
+    }
+
+}
+
+library ForeignControllerDeploy {
+
+    function deploy(
+        address admin,
+        address psm,
+        address usdc,
+        address cctp
+    )
+        external returns (address almProxy, address foreignController, address rateLimits)
```

Comment:
I would use a struct for this instead of a tuple as it's much easier to manage. Like this: https://github.com/makerdao/usds/blob/master/deploy/UsdsDeploy.sol#L37

Reply:
Updated